app/helpers/groups_helper.rb

```diff
@@ -22,8 +22,12 @@ def get_students_table_info
   # Gets information needed to display group info
   # on the front-end. Attributes include URLs for actions
   # such as validation, renaming, showing notes, etc.
-  def get_groupings_table_info
-    groupings = @assignment.groupings
+  def get_groupings_table_info(target_assignment=nil)
```

Comment:
Thoughts on this refactor?

My idea was to make it so previous functions remained untouched, but there's an option in the case where I want to pass in the parent assignment reference from the peer review assignment, without doing the
`@assignment = @assignment.parent_assignment` hack, which I'm assuming is not good in case there are unknown side effects I'm not aware of.